src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -356,6 +356,23 @@ export default class PhotoField extends React.Component {
     this.refs.cropper.zoomTo(e.target.value);
   }
 
+  onZoomMouseMove = (e) => {
+    if (this.state.mouseDown) {
+      this.onZoomSliderChange(e);
+    }
+  }
+
+  onZoomMouseDown = (e) => {
+    this.setState({ mouseDown: true });
```

Comment:
I don't think you need to use `setState`, I think you can make it an instance property (`this.mouseDown = true`). Re-rendering on these events isn't necessary, as far as I can tell.

Reply:
Thanks, that's better, changed.